backend/controllers/principle-controller.js

```diff
@@ -14,7 +17,23 @@ const registration = async (req, res) => {
                 password
             );
 
-            return res.status(201).json(principleData);
+            const emailVerificationToken = jwt.sign(
+                {
+                    email,
+                },
+                process.env.JWT_EMAIL_VERIFICATION_SECRET,
+                { expiresIn: "1h" }
+            );
+
+            await emailService.sendActivationEmail(
+                email,
+                emailVerificationToken
+            );
+            return res.status(201).json({
+                message: `user ${principleData.email} registered, verification link sent`,
```

Comment:
what happens to this message when we add french?

Reply:
I think we do, we have to communicate back to a user that they have to check an email and verify the account/email. (Sorry, I didnt use the same wording from the screenshot, that can be changed) Or do we create this message in the front upon registering a user? 
![Screen Shot 2022-12-19 at 8 51 33 AM](https://user-images.githubusercontent.com/55157764/208453131-33306cd3-1e1f-4816-8782-9159a5313e8e.png)